subnet/kube/kube.go

```diff
@@ -138,7 +139,18 @@ func newKubeSubnetManager(c clientset.Interface, sc *subnet.Config, nodeName, pr
 	ksm.client = c
 	ksm.nodeName = nodeName
 	ksm.subnetConf = sc
-	ksm.events = make(chan subnet.Event, 5000)
+	scale := 5000
+	scaleStr := os.Getenv("K8S_SCALE")
```

Comment:
How about EVENT_QUEUE_DEPTH instead  of K8S_SCALE?
We also want this env var documented. Here: https://github.com/flannel-io/flannel/blob/master/Documentation/configuration.md#environment-variables